bip-taproot.mediawiki

```diff
@@ -88,10 +87,11 @@ The following rules only apply when such an output is being spent. Any other out
 **** If ''k<sub>j</sub> &ge; e<sub>j</sub>'': ''k<sub>j+1</sub> = hash<sub>TapBranch</sub>(e<sub>j</sub> || k<sub>j</sub>)''.
 ** Let ''t = hash<sub>TapTweak</sub>(bytes(P) || k<sub>m</sub>) = hash<sub>TapTweak</sub>(2 + (c[0] & 1) || c[1:33] || k<sub>m</sub>)''.
```

Comment:
I think the `(c[0] & 1)` component needs to be dropped here, or Q would need to be computed using a dependency on itself. It's also inconsistent with the definition of `bytes(P)` now.

Reply:
also doesn't match the python code. fixed